package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@dsnp/activity-content",
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/index.js",
+  "types": "dist/types/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/LibertyDSNP/activity-content.git"
+  },
+  "license": "Apache-2.0",
+  "scripts": {
+    "build:cjs": "tsc -p tsconfig.json",
+    "ts-types": "tsc --emitDeclarationOnly --outDir ./dist/types",
+    "build": "mkdir -p ./dist/types/ && esbuild --format=cjs ./src/index.ts --bundle --outdir=./dist --outbase=src && npm run ts-types",
+    "clean": "rm -Rf dist ./*.d.ts ./*.js ./*.tgz",
+    "doc": "rm -Rf dist/docs && npm run doc:markdown",
+    "doc:html": "typedoc --options ./typedoc.json --plugin none --theme default --out dist/docs",
+    "doc:config": "typedoc --showConfig --options ./typedoc.json --plugin none --theme default --out dist/docs",
+    "doc:json": "typedoc --options ./typedoc.json --json dist/docs/typedoc.json",
+    "doc:markdown": "typedoc --options ./typedoc.json --theme markdown --out dist/docs",
+    "format": "eslint \"**/*.{js,ts}\" --fix",
+    "lint": "eslint \"**/*.{js,ts}\"",
+    "pre-publish": "cp package.json dist",
+    "pub": "cd dist & npm publish",
+    "postpub": "cd dist & del package.json",
+    "pack": "npm run pre-publish && cd dist && npm pack && mv *.tgz ../",
```

Comment:
I didn't notice this until now, but we should talk about this part. This is using the pattern of making a package.json specifically for release. I don't think we need that, but it would be nice in some ways. Either way, it changes how we publish the package and the entry points and such.